exshome.exs

```diff
@@ -0,0 +1,35 @@
+root_folder = "HOME" |> System.get_env() |> Path.join(".exshome")
```

Comment:
please change this to `tmp/exshome`. I'd rather not have these scripts have any footprint but if they need to, keep it in this dir.